components/humantask/humantask-explorer-web/src/web/template/statsView.jag

```diff
@@ -0,0 +1,94 @@
+<div class="col-sm-12">
+  <div class="row">
+    <% if(readyTaskList.length() > 0 || completedTaskList.length() > 0 || reservedTaskList.length() > 0 ||
```

Comment:
These lists gets initialized in the statModel.jag (https://github.com/wso2/carbon-business-process/blob/master/components/humantask/humantask-explorer-web/src/web/model/statsModel.jag#L80) and if the lists are empty it will not be undefined or null but it will be an xml array with length 0.